source/tests/system/nirfmxbluetooth_driver_api_tests.cpp

```diff
@@ -30,8 +30,9 @@ namespace tests {
 namespace system {
 namespace {
 
-const auto kPxi5663e = "5663E";
-const int kPreambleSyncPacketStartDetectionFailedWarning = 686280;
+constexpr auto PXI_5663E = "5663E";
+constexpr auto PREAMBLE_BURST_SYNC = 686280;  // The Preamble Sync failed to detect start of the packet
+constexpr auto TYPE_MISMATCH_ERROR = -380251;
```

Comment:
We should be consistent in our constants some use this _ERROR suffix while others don't. 

I vote for all having either "_ERROR" or "_WARNING" suffix